app/controllers/volunteers_controller.rb

```diff
@@ -1,8 +1,8 @@
 class VolunteersController < ApplicationController
   include SmsBodyHelper
 
-  before_action :set_volunteer, except: %i[index new create datatable stop_impersonating]
-  after_action :verify_authorized, except: %i[stop_impersonating]
+  before_action :set_volunteer, except: %i[index new create datatable stop_impersonating table_state table_state_update]
+  after_action :verify_authorized, except: %i[stop_impersonating table_state table_state_update  table_state]
```

Comment:
we will need to make sure there is authorization on this (we have the current user to use).

Reply:
since we're suing the preference_sets controller this is not relevant anymore, but I'm skipping authorization in the new controller.  I didn't think it needs it since this methods don't render a view and only seems to be accessible by the appropriate user.
I can add it if needed